enable/qt/base_window.py

```diff
@@ -461,12 +461,17 @@ def _create_mouse_event(self, event):
                 y = event.position().y()
             modifiers = event.modifiers()
             buttons = event.buttons()
+        # The AttributeError is usually trigged when the mouse pointer
+        # leaves the ui window since the event of leaving the window
+        # is a "QEvent", which doesn't contain x and y positions like
+        # QMouseEvent (for mouse movement) or QEnterEvent (for mouse
+        # entering the window) from QGui.
```

Comment:
This is a somewhat misleading comment (whether this path gets triggered is unrelated to whether the event is defined in `QtGui`) and the reasoning is covered by the comment in lines 453-454.

Reply:
Sure, I will remove it. We came up with this comment during my standup with Didrik and thought it may be useful since it shows how the attribute error can happen (by moving the mouse out of the window). What if I keep only this part:

```
        # The AttributeError is usually trigged when the mouse pointer
        # leaves the ui window since the event of leaving the window
        # is a "QEvent", which doesn't contain x and y positions
``` ?